cmd/edit/ingress/cmd.go

```diff
@@ -155,6 +175,28 @@ func run(cmd *cobra.Command, argv []string) {
 		private = &privArg
 	}
 
+	var lbType *string
+	if cmd.Flags().Changed(lbTypeFlag) {
```

Comment:
Same NP

Reply:
Using the Changed() flag is the pattern used for the other two parameters in the same function (see above these lines) so I retained the same pattern. Should I change it ? and just for this parameter ?

Reply:
Oh no, if that's the case let's keep it that way

Reply:
Also, another issue I found is that if we don't check the flag change, then even if the user provides the desired lbType as a flag and has the interactive flag also set, then it ignores the command line flag which is not good. So I think it may be best to retain the Changed() flag for both create and edit functions and that seems to work better in testing.